### DISCUSSION TITLE
Codespaces Config for using Antares

### DISCUSSION THREAD

**Discussion by tim-breitenbach** [AntaresSimulatorTeam/Antares_Simulator#1292] [Ideas]
Hey everyone,

in our testing phase of Antares@Swissgrid we made Antares work in a GitHub Codespace.
With this we can make sure that everyone running Antares@Swissgrid can have it available in an instant(we prebuild the container images daily so startup times are fast(<30secs)) and on the machine size they want.
You basically just have to copy the .devcontainer directory in the repo and Github can pick it up as a Codespace Image.

Things we have archieved:
1. Made Antares run in a Codespace Container
2. Made Antares run on Ubuntu 22.04 LTS(atleast the features we use, so far we have't encountered something that behaves weird)

Would providing this Codespace Config plus UpdateContent Shell Scripts be useful for you?
We are happy to provide that for your documentation or somewhere else.
I thought it doesn't make sense to da a PR on the Repo since its more for people using Antares than really developing on its core.

We are still figuring out if Codespaces is the place to run big simulations with it but for small ones it is definitly good.

BR,
Tim

**Comment by flomnes** [comment]
Hello @tim-breitenbach, thanks for your suggestion, sorry for the late reply. We are always eager to welcome new contributions & making Antares Simulator easier to deploy.

Don't hesitate to open a PR.

Florian
